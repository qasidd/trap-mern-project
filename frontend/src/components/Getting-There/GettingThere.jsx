'use strict';

import GoogleMap from "./GoogleMap";

const GettingThere =()=>{
    return(
        <>
        <section>
            <div className="container text-center">
            <h2>Cinema Location </h2>
            </div>
        </section>
        <GoogleMap/>


        </>
    )
}

export default GettingThere;