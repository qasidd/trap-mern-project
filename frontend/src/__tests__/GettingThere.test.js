import GettingThere from '../components/Getting-There/GettingThere';
import '@testing-library/jest-dom';
import { expect, it } from '@jest/globals';
import { create } from 'react-test-renderer';

it(`It should match the snapshot every time it's rendered`, () => {
    const instance = create(<GettingThere/>).toJSON();
    expect(instance).toMatchSnapshot();
    
})