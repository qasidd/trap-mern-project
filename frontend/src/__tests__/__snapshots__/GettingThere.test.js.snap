// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should match the snapshot every time it's rendered 1`] = `
Array [
  <section>
    <div
      className="container"
    >
      <div
        className="text-center"
      >
        <h2>
          Getting There
        </h2>
      </div>
    </div>
  </section>,
  <br />,
  <section>
    <div
      className="ui grid"
    >
      <div
        className="row"
      >
        <div
          className="eight wide column"
        >
          <div
            className="container text-center"
          >
            <img
              alt="QA Cinemas"
              height="450px"
              src="QA.jpg"
              width="650px"
            />
          </div>
          <div
            className="text-center"
          >
            <footer>
              <h6>
                Image of QA Cinemas
              </h6>
            </footer>
          </div>
        </div>
        <div
          className="eight wide column"
        >
          <div
            className="container text-center"
          >
            <iframe
              allowfullscreen=""
              height="450"
              loading="lazy"
              src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2374.69070876321!2d-2.2885056841567497!3d53.473987480006045!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x487bae15ae0912e7%3A0x39957d91b0a3996f!2sQA!5e0!3m2!1sen!2suk!4v1614871582257!5m2!1sen!2suk"
              style={
                Object {
                  "border": 0,
                }
              }
              width="650"
            />
            <div
              className="container text-center"
            >
              <footer>
                <h6>
                  Location Map of the cinema
                </h6>
              </footer>
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>,
  <br />,
  <section>
    <div
      className="card"
    >
      <h6>
        Address 1:
      </h6>
      <p>
        5th Floor Anchorage, 1 Anchorage Quay, Salford Quays, Manchester, M50 3YJ
      </p>
      <h6>
        Telephone:
      </h6>
      <p>
        01942 527 010
      </p>
      <h6>
        Address 2:
      </h6>
      <p>
        3 Anchorage Quay, Salford, M50 3XL
      </p>
    </div>
    <br />
    <div>
      <h4>
        Public Transport Routes 
      </h4>
    </div>
    <div
      className="card"
    >
      <h6>
        Tram Directions
      </h6>
      <ul>
        Directions from the city center:
        <li>
          Enter the tram from either Piccadilly Station or St Peter Square.
        </li>
        <li>
          Please select the Blue line or Orange line link towards Media City UK or Eccles.
        </li>
        <li>
          25 minute ride from the City Centre to the Anchroage Station.
        </li>
        <li>
          If you are traveling from Deansgate/Castlefield it's a 15 minute tramp ride with 5 stops until you get to the Anchorage tram station.
        </li>
        <li>
          For Other Routes: Please exchange at Cornbrook Station, if you are traveling from East Didsbury, Trafford, Altrincham or Chorlton to select the correct line.
        </li>
      </ul>
    </div>
    <br />
    <div
      className="card"
    >
      <h6>
        Bus Directions:
      </h6>
      <ul>
         Bus Routes from the City Centre:
        <li>
          Take the bus 33 toward Worsely
        </li>
        <li>
          A 25 minute bus ride to the stop on Howard Street, then a 5-10 minute walk to reach the cinema.
        </li>
        <li>
          Other buses that travel here: 33, 29, 79, 53 and 50.
        </li>
      </ul>
    </div>
    <br />
    <div
      className="card"
    >
      <h5>
        Car Park Details
      </h5>
      <h6>
        Parking Center Address:
      </h6>
      <p>
        Anchorage, Anchorage Quay, Salford, Quays M50 3XE
      </p>
      <p>
        10 minute walk from the cinema
      </p>
      <p>
        There's no car parking for delegates, the nearest carpark is U Swim Carpark. However there's 10 spaces for permanent staff, call 01613932270 and ask for Hannah who would be happy to assist.
      </p>
    </div>
  </section>,
]
`;
